src/fees/compute_fees.py

```diff
@@ -21,17 +21,39 @@
 class Trade:
     """Class for"""
 
-    order_uid: HexBytes
-    sell_amount: int
-    buy_amount: int
-    sell_token: HexBytes
-    buy_token: HexBytes
-    limit_sell_amount: int
-    limit_buy_amount: int
-    kind: str
-    sell_token_clearing_price: int
-    buy_token_clearing_price: int
-    fee_policies: list["FeePolicy"]
+    def __init__(
```

Comment:
a `__post_init__` method could be used instead to set the remaining fields which require computation.